flw/src/main/java/org/motechproject/nms/flw/domain/FlwError.java

```diff
@@ -0,0 +1,62 @@
+package org.motechproject.nms.flw.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Audit log object for tracking failed FLW updates
+ */
+@Entity(tableName = "nms_flw_errors")
```

Comment:
Why is this table unique to this ops endpoint?  Do we have similar logging for the web services import?  (will I find that later in the code review?)